xml/deploy_cephadm.xml

```diff
@@ -68,7 +68,8 @@
   <procedure>
    <step>
     <para>
-     Install and register &cephos; on each cluster node. Include at least the
+     Install and register &cephos; on each cluster node. During installation of &productname;,
+      access to the update repositories is requiered, therefore registering is mandatory. Include at least the
```

Comment:
Typo: "requiered" (required).

Sorry, I did not see this earlier...

Reply:
@lproven , oops - will you sent a followup change or shall I?

Reply:
I'll do it myself so you can review ;)

Reply:
I see it's fixed already - thanks!